sdk/python/feast/infra/firestore.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+from feast import FeatureTable
+from feast.infra.provider import Provider
```

Comment:
also a bit of a nitpick, but does `infra` add value here? What do we lose if we remove `infra` as a submodule?

Reply:
i wanted all cloud specific stuff to be in a subpackage, could rename the subpackage to "provider" too

Reply:
I guess its not a big deal for now. let's just leave it as is.